### DISCUSSION TITLE
Consult crazyflie's physical parameters

### DISCUSSION THREAD

**Discussion by Admire-ljb** [bitcraze/discussions#23] [Q&A] [answered]
We want to use the trajectory planning algorithm to send the full state command to crazyFlie, but it always falls. Is our speed acceleration set too high? May I ask where can we see the kinematic parameters of Crazyflie, e.g. max_acc, max_vel, etc.

**Comment by whoenig** [comment, marked as answer]
I use 2m/s^2 and 0.5m/s, but this is pretty conservative.

**Comment by Admire-ljb** [comment]
Thank you. I'll give it a try
